core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-box/template-builder-box.component.html

```diff
@@ -0,0 +1,59 @@
+<ng-template [ngIf]="size !== 'small'">
```

Comment:
There is ngif/else for example:

```
<div *ngIf="isLoggedIn; else loggedOut">
  Welcome back, friend.
</div>

<ng-template #loggedOut>
  Please friend, login.
</ng-template>
```
More info: https://ultimatecourses.com/blog/angular-ngif-else-then